Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,51 @@
+.container {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  margin: 30px 0 90px 120px;
```

Comment:
the 30px should be achieved by a margin bottom in the title. I'd add the same margin to both sides in order to have a consistent box. I'd lower the 120px too because that might be a very big value